utilities/metadata_parser.py

```diff
@@ -215,13 +210,27 @@ def interpret_person_organization(field, item_organization, item_person, new_doc
     field_split = [person_or_organization.strip() for person_or_organization in field.split(';')]
 
     for person_or_organization in field_split:
+
+        # skip useless / empty values
+        if person_or_organization in {'', 'none', 'None', 'unknown',
+                                      'Multiple', 'copy of above'}:
+            continue
+
+        # if no comma -> likely organization
         if len(person_or_organization.split(',')) == 1:
+
+            # spell out ONR for the different offices
+            person_or_organization = person_or_organization.replace('ONR', 'Office of Naval '
+                                                                           'Research')
+
             new_org, _unused_org_created = Organization.objects.get_or_create(
                 name=field_split[0],
```

Comment:
This is the problem line... that's always pointing at the first name in the list.